### ISSUE TITLE
Edit case number of existing time slot

### ISSUE THREAD

**Issue by thomasguerena** [jomogalla/lister#6]
Right now you can edit start and end times of an existing time slot, but there isn't a way to change the case number in-place (leaving the time slot as-is).

![image](https://user-images.githubusercontent.com/2658810/35174858-c19c1558-fd25-11e7-96d5-a89c2083ead8.png)

Suggested fix:
- Rather than clicking the case number link to open case details, allow users to click any part of the row.
- Change the case number to be in-line editable, like start and end times.

**Comment by jomogalla** [comment]
me gusta

**Comment by jomogalla** [comment]
I know this isnt as easy, but offers the same functionalty - you could use the add time with a different case, same times and I *think* it would replace the case

**Comment by thomasguerena** [comment]
Yeah, that does work. I was basically trying to avoid copying times into that form below. Also, a future feature I think could be nice to have in this app (separate feature request) would be _ctrl-z_ support. I always likes apps that auto-save to also let me undo accidental changes.
